gulpfile.js

```diff
@@ -142,9 +142,12 @@ gulp.task("sonarqube", callback => {
     });
   } else if (process.env.TRAVIS_PULL_REQUEST !== "false") {
     runSonnarQubeScanner(callback, {
+      "sonar.pullrequest.branch": process.env.TRAVIS_PULL_REQUEST_BRANCH,
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace double quotes by simple quote ([javascript:SingleQuote](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3ASingleQuote))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarlint.vscode%3Asonarlint-vscode&pullRequest=16&issues=AWMHLCjdaT4gCmfv8i_m&open=AWMHLCjdaT4gCmfv8i_m)

[ISSUE_KEY]: AWMHLCjdaT4gCmfv8i_m